### PR TITLE
Dashboard / Analytics changes from ORUK fork

### LINKED ISSUE #157: [page title] Unable to distinguish content on page
The page title is always "Open Referral UK" and should describe the page content.

URL: https://openreferraluk.org/about-standard

Pattern to use
[H1] - Open Referral UK

### LINKED ISSUE #171: [content flash] Flash of content "404 - content not found!"
URL: https://openreferraluk.org/community/case-studies

When landing on the page users see a flash of content "404 - content not found!". This occurs for every refresh.

https://user-images.githubusercontent.com/2459151/123019473-74b6f780-d3c8-11eb-8274-35fcc13ea220.mov

### PR DESCRIPTION
**Pull request by kingdomlevel**
Significant number of changes from ORUK fork of this repo.
Changes mostly around google analytics and the dashboard component.

### CODE REVIEW

File: src/components/casestudies/LandingPage.js [hunk 1]
```diff
@@ -2,42 +2,48 @@ import { useState, useEffect } from 'react';
...
+	const [{ data: casesData, isError: isCasesError}, isFeatchingCases] = useOukapi(new URL(CASE_STUDIES, BASE_URL).href);
+
+	useEffect(() => {
+		if (landingData.hasOwnProperty("title") && landingData.hasOwnProperty("introParagraph")) {
+			setTitle(landingData.title);
+			setIntroParagraph(landingData.introParagraph)
+		}
+	}, [landingData]);
+
+	if (isFeatchingLanding || isFeatchingCases)
```
Comment (kingdomlevel) [inline src/components/casestudies/LandingPage.js:26]: Th

File: src/components/casestudies/LandingPage.js [hunk 2]
```diff
@@ -2,42 +2,48 @@ import { useState, useEffect } from 'react';
...
+		return null;
```
Comment (kingdomlevel) [inline src/components/casestudies/LandingPage.js:27]: Fixes #171

File: src/components/genericcontentpage/PageTitle.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        return () => {
+            document.title = mainTitle;
+            //window.ga('send', 'pageview', { page: location.pathname, title: document.title });
+        };
+    }, [title, location]);
+
+    return <></>;
+};
+
+export default PageTitle;
```
Comment (kingdomlevel) [inline src/components/genericcontentpage/PageTitle.js:34]: Closes #157 as this component and its usage updates the page title to change dependent on the page we are on.

**Comment by adamgparsons** [comment]
LGTM

**Review by adamgparsons — APPROVED** [review]

**Review by kingdomlevel — COMMENTED** [review]
Looks good! Counting this as a valid review as this is not my work (code from Porism fork)

**Pull request merged**